dropwizard-db/src/main/java/io/dropwizard/db/DataSourceFactory.java

```diff
@@ -429,6 +429,10 @@ public String getValidationQuery() {
         return validationQuery;
     }
 
+    public String getHealthCheckValidationQuery() {
```

Comment:
I think we should use `@Override` on the implementations of `PooledDataSourceFactory`